apps/AEPSampleAppNewArchEnabled/README.md

```diff
@@ -1,50 +1,93 @@
 
-## Get started
 
-1. Install dependencies
+## Introduction
+Welcome to the **Expo AEPSample App with New Architecture Enabled**. This project is an [Expo](https://expo.dev) app created using [`create-expo-app`](https://www.npmjs.com/package/create-expo-app). It demonstrates the integration of Adobe SDKs with a React Native app using Expo, with the new architecture enabled by default.
 
-   ```bash
-   npm install
-   ```
+## Prerequisites
+Ensure that your environment meets the following requirements before running this project:
 
-2. Start the app
+- Familiarity with [React Native development environment setup](https://reactnative.dev/docs/environment-setup) is recommended (preferably with the React Native CLI Quickstart).
+- **Cocoapods**: [Version 1.11.3 or higher](https://guides.cocoapods.org/using/getting-started.html).
+- **Node.js**: [Version 18 or higher](https://nodejs.org/en/download/).
+- **React Native**: [Tested with version 0.75.x](https://reactnative.dev/).
+- **Expo**: [Tested with version 51](https://docs.expo.dev/).
+- Note: The new architecture is enabled by default in this build.
 
-   ```bash
-    npx expo start
-   ```
 
-In the output, you'll find options to open the app in a
+## Installation
+To install **AEPSample App**, follow these steps:
 
-- [development build](https://docs.expo.dev/develop/development-builds/introduction/)
-- [Android emulator](https://docs.expo.dev/workflow/android-studio-emulator/)
-- [iOS simulator](https://docs.expo.dev/workflow/ios-simulator/)
-- [Expo Go](https://expo.dev/go), a limited sandbox for trying out app development with Expo
+### Install dependencies
+> Note: Commands are assuming you're in the root directory of the repository.
 
-You can start developing by editing the files inside the **app** directory. This project uses [file-based routing](https://docs.expo.dev/router/introduction).
-
-## Get a fresh project
+```
+yarn install && yarn sampleapp:ios:pod:install
+```
 
-When you're ready, run:
+### Build packages
 
-```bash
-npm run reset-project
 ```
+yarn run build
+```
+
+## Get Started
+
+### Configure the App ID
+
+- **iOS**: In `apps/AEPSampleAppNewArchEnabled/ios/AEPSampleAppNewArchEnabled/AdobeBridge.m`, find the `configureWithAppId` call and add your app ID.
+- **Android**: In `apps/AEPSampleApp/android/app/src/main/java/com/aepsampleapp/MainApplication.kt`, find the `configureWithAppId` call and add your app ID.
+
+> **Note**: The App ID should be configured in a mobile property in the Data Collection UI, in Tags. Refer to [Adobe documentation](https://developer.adobe.com/client-sdks/home/base/mobile-core/configuration/#configure-with-app-id-per-environment) for more information.
+
+## Run Instructions
 
-This command will move the starter code to the **app-example** directory and create a blank **app** directory where you can start developing.
+### Run Instructions for iOS
+1. Ensure you are in the root directory.
+   ```bash
+   yarn sampleappnewarchenabled:ios:run
```

Comment:
did we do this already in the previous steps?  We just need one of them.

Reply:
In earlier steps we have just installed dependencies for iOS but have not run the app